code/modules/shuttle/supply.dm

```diff
@@ -69,8 +69,8 @@ GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 		var/area/shuttle/shuttle_area = place
 		for(var/trf in shuttle_area)
 			var/turf/T = trf
-			for(var/a in T.GetAllContents())
-				if(is_type_in_typecache(a, GLOB.blacklisted_cargo_types) && !istype(a, /obj/docking_port))
+			for(var/item in T.GetAllContents())
```

Comment:
since you changed `a`, can't you also change the `T`?

Reply:
Sure

Reply:
Also changed it to use as anything too